chapter_computer-vision/fine-tuning.md

```diff
@@ -1,12 +1,12 @@
 # 微调
 :label:`sec_fine_tuning`
 
-在前面的一些章节中，我们介绍了如何在只有6万张图像的Fashion-MNIST训练数据集上训练模型。
-我们还描述了学术界当下使用最广泛的大规模图像数据集ImageNet，它有超过1,000万的图像和1,000类的物体。
+在前面的一些章节中，我们介绍了如何在只有 6 万张图像的 Fashion-MNIST 训练数据集上训练模型。
+我们还描述了学术界当下使用最广泛的大规模图像数据集 ImageNet，它有超过 1,000 万的图像和 1,000 类的物体。
```

Comment:
Sorry for a bit late reply. Please don't add a space to the text, please see more details in https://github.com/d2l-ai/d2l-zh/pull/868 and https://github.com/d2l-ai/d2l-zh/pull/889. Thanks!